portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,50 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+import org.json.simple.JSONObject;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/comment")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> allComments = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    if (allComments.size() == 0) {
+        allComments.add("Welcome to my mailbox :)");
+        allComments.add("Hi, I am Yi Lyu. My foreign name is Isabella :D");
+        allComments.add("I am looking forward to know u.");
+        allComments.add("You can contact me with the info in the left side bar!");
+    }
```

Comment:
This is duplicated with what you have in the doPost(), consider make it a function?